llvm/test/lab2/ryabkov_vladislav/tester.ll

```diff
@@ -0,0 +1,149 @@
+; RUN: opt -load-pass-plugin %llvmshlibdir/RyabkovInstrumentFunctions%pluginext -passes=instrumentation-functions -S %s | FileCheck %s
```

Comment:
Please, check your solution on this IR: https://godbolt.org/z/Kaazc4Gao

and add similar test

Reply:
done